BBE_Generation/.gitignore

```diff
@@ -0,0 +1,63 @@
+# Compiled class file
```

Comment:
Please keep this content inside publish tools directory and avoid a separate gitignore file and stick to mainone